modules/remind.py

```diff
@@ -109,10 +79,48 @@ def monitor(jenni):
     'sec': 1,
     's': 1
 }
+r_command = None
+
+
+def setup(jenni):
+    global r_command
+
+    periods = '|'.join(scaling.keys())
+    p_command = r'{}in ([0-9]+(?:\.[0-9]+)?)\s?((?:{})\b)?:?\s?(.*)'.format(
+        jenni.config.prefix,
+        periods,
+    )
+    r_command = re.compile(p_command)
+
+    jenni.rfn = filename(jenni)
+
+    # jenni.sending.acquire()
+    jenni.rdb = load_database(jenni.rfn)
+    # jenni.sending.release()
+
+    def monitor(jenni):
+        time.sleep(5)
+        while True:
+            now = int(time.time())
+            unixtimes = [int(key) for key in jenni.rdb]
+            oldtimes = [t for t in unixtimes if t <= now]
+            if oldtimes:
+                for oldtime in oldtimes:
+                    for (channel, nick, message) in jenni.rdb[oldtime]:
+                        if message:
+                            jenni.msg(channel, nick + ': ' + message)
+                        else: jenni.msg(channel, nick + '!')
+                    del jenni.rdb[oldtime]
+
+                # jenni.sending.acquire()
+                dump_database(jenni.rfn, jenni.rdb)
+                # jenni.sending.release()
+            time.sleep(2.5)
+
+    targs = (jenni,)
```

Comment:
That is how singleton tuples are suppose to be constructed, see https://docs.python.org/2/tutorial/datastructures.html#tuples-and-sequences.

`threading.Thread`'s args argument accepts tuples to pass into its `__init__` function which is then unpacked and sent into the `monitor` function as arguments.